cameraDemo/src/main/java/com/visenze/visearch/camerademo/views/adapter/StrechImageAdapter.java

```diff
@@ -78,8 +78,10 @@ public View getView(int position, View convertView, ViewGroup parent) {
                 .tag(mContext)
                 .into(viewHolder.imageView);
 
-        int score = (int)(imageList.get(position).getScore() * 100);
-        viewHolder.scoreView.setText("Similarity " + String.valueOf(score) + "%");
+        Float score = imageList.get(position).getScore();
+        if (score != null) {
+            viewHolder.scoreView.setText("Similarity " + String.valueOf((int)(score * 100)) + "%");
+        }
 
         return imageLayout;
     }
```

Comment:
@swVisenze how come there are Apache HTTP client and visearch android JAR under dependency folder? can you remove ?

Reply:
this pr is to remove the jar files under dependency folders. all jar files in depencency folder is removed .

Reply:
oh I see, thanks, i thought it is added